src/locales/en/translation.json

```diff
@@ -348,16 +349,23 @@
     "Warsztat 1": "Workshop 1",
     "Warsztat 2": "Workshop 2",
     "Warsztat 3": "Workshop 3",
-    "Warsztat 4": "Workshop 4"
+    "Warsztat 4": "Workshop 4",
+    "traditional": "Traditional menu",
+    "vegetarian": "Vegetarian menu"
   },
   "checkboxComponent": {
-    "label": "I consent to the processing of my personal data."
+    "certificate": "I would like to receive a certificate of participation in the Attention! Smartphone conference.",
+    "consent": "I consent to the processing of my personal data."
   },
   "workshopsForm": {
     "header": "Workshops Registration Form",
-    "paragraph": "Each participant of the event Uwaga! Smartfone during the conference has the opportunity to participate in one selected workshop. Please familiarize yourself with the program and choose one of the workshops below. Thank you.",
+    "paragraph": "Each participant of the event Attention! Smartphone during the conference has the opportunity to participate in one selected workshop. Please familiarize yourself with the program and choose one of the workshops below. Thank you.",
```

Comment:
Wydaje mi się, że nie powinniśmy w wersji ang zostawić oryginalną nazwę :D ale nie wiem jaki mam przykaz "z góry" na to ;g